Sources/Sentry/SentryScope.m

```diff
@@ -59,26 +59,33 @@ @interface SentryScope ()
  */
 @property(atomic) enum SentryLevel levelEnum;
 
+@property(atomic) NSInteger maxBreadcrumbs;
+
 @end
 
 @implementation SentryScope
 
 #pragma mark Initializer
 
-- (instancetype)init {
+- (instancetype)initWithMaxBreadcrumbs:(NSInteger)maxBreadcrumbs {
```

Comment:
Should be called by whoever instantiates it since it holds SentryOptions with Maxbreadcrumb